portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/storage/JobStore.java

```diff
@@ -74,21 +75,21 @@
    */
   UUID findFirst(JobAuthorization.State jobState);
 
-  default <T extends DataModel> void create(UUID jobId, String key, T model) throws IOException {
+  default <T extends Serializable> void create(UUID jobId, String key, T model) throws IOException {
```

Comment:
Hmm, do we need to know the type is a DataModel vs arbitrary data?

Reply:
I don't think so, it hasn't been a problem in my testing at all.